Xamarin.Essentials/Platform/Platform.macos.cs

```diff
@@ -20,6 +23,46 @@ internal static NSWindow GetCurrentWindow(bool throwIfNull = true)
         }
     }
 
+    internal static class ApplicationServices
+    {
+        const string appServicesPath = "/System/Library/Frameworks/ApplicationServices.framework/Versions/Current/ApplicationServices";
+
+        // https://developer.apple.com/documentation/coregraphics/1454595-cgdisplaycreateimage
+        [DllImport(appServicesPath, EntryPoint = "CGDisplayCreateImage")]
+        static extern /* CGImageRef */ IntPtr CGDisplayCreateImage(int displayId);
```

Comment:
Can we use something derived from SafeHandle here? Would be preferable to oblique IntPtrs and ease freeing of resources

Reply:
I think this should be discussed in separate issue. `Platform.IOKit` would also be affected.